static/js/formatters-internal.js

```diff
@@ -442,33 +384,21 @@ function _getEuImageDynamicUrl(image, desiredSize, atLeastAsLarge) {
 
   formatOptions.push(`fit=${atLeastAsLarge ? 'cover' : 'contain'}`);
```

Comment:
I realize this is existing code, but is it right? There's no exact equivalent, but I think it might be better to use `scale-down` for both (which is the default). Maybe `contain` for least would be more correct, but dynl doesn't enlarge.

But the other options mess with the aspect ratio (`cover` crops it), which dynl/dynm never do, so that doesn't seem right. See docs:
https://developers.cloudflare.com/images/transform-images/transform-via-workers/#fit

Reply:
Talked to @nmanu1 offline. The expected behavior would be:
- if both dims (50*100 or 200*450) provided and original image size were 100x150
    - atLeastAsLarge=true, return 66x100 or 300x450
    - atLeastAsLarge=false, return 50x75 or 200x300
- if one dim (width=50 or 200) provided and if the original image size were 100x150, return 50x75 or 200x300 regardless of atLeastAsLarge
- if no dims provided, return original size

Updated the code to follow that.